contrib/python/test/test_images.py

```diff
@@ -81,9 +82,15 @@ def test_get(self):
         self.assertEqual(actual, self.alpine_image)
 
     def test_history(self):
-        for count, record in enumerate(self.alpine_image.history()):
-            self.assertEqual(record.id, self.alpine_image.id)
-        self.assertGreater(count, 0)
+        records = []
+        bucket = Counter()
+        for record in self.alpine_image.history():
+            self.assertIn(record.id, (self.alpine_image.id, '<missing>'))
+            bucket[record.id] += 1
+            records.append(record)
+
+        self.assertGreater(bucket[self.alpine_image.id], 0)
+        self.assertEqual(sum(bucket.values()), len(records))
```

Comment:
Do we need this?  It looks like "Is `Counter` working?".

Reply:
I wanted to verify nothing got dropped.